packages/dom-expressions/src/server.js

```diff
@@ -667,7 +667,19 @@ export function ssrSpread(props, isSVG, skipChildren) {
     ) {
       continue;
     } else {
+      // bool:
+      if (prop.slice(0, 5) === "bool:") {
+
+       if(!value) continue
+       prop = prop.slice(5);
+       result += `${escape(prop)}`;
```

Comment:
does this need a space?

Reply:
@ryansolid this has been merged, pinging because Im not sure if you saw the question above, thanks

Reply:
Yeah it can but it depends which is why we have that logic after the condition. I'm going to slightly refactor to include it.